.github/workflows/test-job.yaml

```diff
@@ -109,6 +111,31 @@ jobs:
       run: make test
       working-directory: ${{env.working-directory}}
 
+    - name: Install Kind
+      run: |
+        curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.11.1/kind-linux-amd64
+        chmod +x ./kind
+        sudo mv ./kind /usr/local/bin/kind
+
+    - name: Checkout Python
+      uses: actions/checkout@v2
+
+    - name: Setup Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install Python dependencies
+      run: pip install -r ./tests/config/requirements.txt
+
+    - name: Run compatibility test
+      run: |
+        for i in "rayproject/ray:1.8.0" "rayproject/ray:1.9.0" "rayproject/ray:1.10.0"
```

Comment:
let's use 1.8.0 1.9.0 to emphasize versions, KUBERAY_TEST_RAY_IMAGE=rayproject/ray:${version}?